edgelet/contrib/centos/aziot-edge.spec

```diff
@@ -89,6 +91,11 @@ if /usr/bin/getent group systemd-journal >/dev/null; then
     %{_sbindir}/usermod -aG systemd-journal %{iotedge_user}
 fi
 
+# Create an edgeagentuser and add it to iotedge group
+if ! /usr/bin/getent passwd %{iotedge_agent_user} >/dev/null; then
+    %{_sbindir}/useradd -g %{iotedge_group} -c "edgeAgent user" -ms /bin/bash -u %{iotedge_agent_uid} %{iotedge_agent_user}
```

Comment:
Login shell should be `/bin/nologin` just like the `iotedge` user. There's no need for anyone to log in as `edgeagentuser`.

Reply:
/sbin/nologin you mean? Fixed

Reply:
I said `/bin/login` because that's what the `iotedge` user above uses.

Reply:
You're right, let's be consistent.